config/settings.d/dns_nsupdate.yml.example

```diff
@@ -0,0 +1,6 @@
+---
+# Can be true, false, or http/https to enable just one of the protocols
```

Comment:
would this remain true for providers?  Do providers obey enabled/disabled (and what happens if you set :dns_provider to a disabled provider?)